docs/guide/search.md

```diff
@@ -365,9 +365,17 @@ Secondly, copy configuration files needed for eZ Solr Search Engine bundle, in t
 
 ``` bash
 # Make sure to replace the /opt/solr/ path with where you have placed Solr
-cp -R vendor/ezsystems/ezplatform-solr-search-engine/lib/Resources/config/solr/* /opt/solr/example/solr/collection1/conf/
-
-/opt/solr/bin/solr start -f 
+cd /opt/solr/example
```

Comment:
`/opt/solr` is used as an example path in `Solr 6` documentation (one section below one you're changing) so maybe stick to that one?